tests/utils/streams/backends/test_kafka.py

```diff
@@ -9,20 +9,20 @@
 
 import pytest
```

Comment:
I think we should move the tests as well in their own module so they go into the library.

Reply:
I'm going to handle the tests as a follow up.